README.md

```diff
@@ -141,6 +142,23 @@ Sentry as part of the extra data, if it was the last action before an error.
 
 This option was introduced in version 1.1.1.
 
+#### `getUserContext` _(Optional Function)_
+
+Signature: `state => userContext`
+
+The [user context] is an important part of the error reporting. When
+`getUserContext` is a truthy function, the result of `getUserContext`
```

Comment:
All functions are truthy.

Reply:
Yeah. When setting the function to `optional`, using truthy doesn't make that much sense.

Reply:
Maybe just “when `getUserContext` is specified”?

Reply:
:+1: 
Did a small rephrase now :smile: